support/cas-server-support-thymeleaf/src/main/resources/messages.properties

```diff
@@ -621,8 +620,8 @@ cas.mfa.yubikey.label.token=Token:
 cas.mfa.webauthn.auth.fail=Unable to verify your device for authentication. Provided payloads may be invalid, expired or otherwise compromised.
 cas.mfa.webauthn.register.helptext=Assign a friendly-name to your FIDO2-enabled device and then register it with CAS for multifactor \
   authentication. Once device registration is successfully completed, you will automatically \
-  be redirected to the next step to login with your device.
-cas.mfa.webauthn.register.wait=Your FIDO2-enabled device is now registered with CAS. Please wait...
+  be redirected to the next step to log in with your device.
+cas.mfa.webauthn.register.wait=Your FIDO2-enabled device is now registered with CAS. Please wait…
```

Comment:
Are you sure "..." is correct here? The diff is rather strange.

Reply:
Yes, I'm pretty sure. Indeed, the ellipsis can be rendered by this utf-8 char (&hellip; or &mldr;) instead of 3 dots 
see https://practicaltypography.com/ellipses.html

Reply:
As a sample, here is how a string with this char looks in CAS :
<img width="598" alt="Capture d’écran 2024-01-18 à 12 02 17" src="https://github.com/apereo/cas/assets/12731381/86d1baa5-d4c2-448c-b457-f1abe4a13c84">

Reply:
OK. What is the objective behind this change?

Reply:
The horizontal ellipsis is slightly better for typography (see  https://practicaltypography.com/ellipses.html) and for accessibility (a text-to-speech will more easily state "ellipsis" than "dot dot dot")

Reply:
Thanks!